python/lsst/ts/criopy/m1m3/BumpTestTimes.py

```diff
@@ -0,0 +1,112 @@
+# This file is part of M1M3 GUI.
+#
+# Developed for the LSST Telescope and Site Systems.  This product includes
+# software developed by the LSST Project (https://www.lsst.org).  See the
+# COPYRIGHT file at the top-level directory of this distribution for details of
+# code ownership.
+#
+# This program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+# details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+from astropy.time import Time, TimeDelta
+from lsst_efd_client import EfdClient
+
+from ..M1M3FATable import FAType, force_actuator_from_id
+
+
+class BumpTestTimes:
+
+    """Returns a set of times for bump tests given the actuator ID and
+    an input timespan
+
+    Parameters
+    ----------
+    client : lsst_efd_client.efd_helper.EfdClient object
+        This is used to query the EFD
+    """
+
+    def __init__(self, client: EfdClient = EfdClient("summit_efd")):
+        super().__init__()
+        self.client = client
+
+    async def find_times(
+        self,
+        actuator_id: int,
+        start: Time = (Time.now() - TimeDelta(7, format="jd")),
+        end: Time = Time.now(),
+    ) -> tuple[list[float], list[float]]:
+        """Find bump test query times
```

Comment:
Docstring should follow [numpy docs style](https://numpydoc.readthedocs.io/en/latest/format.html):

```
        """Find bump test query times.

        Parameters
        ----------
        id : `int`
            Force Actuator identification number. Starting with 101, the first
            number identified segment (1-4). The value ranges up to 443.

        start : `str`
            Time to begin looking for bump tests.
            Format: 2023-04-17T10:00:00
            Scale: UTC

        end : `str`
            Time to begin looking for bump tests.
            Format: 2023-04-17T10:00:00
            Scale: UTC

        Returns
        -------
        primaryTimes : `list` [ `list` [ `int` ] ]
            Nested list with start/end times of the primary bump tests.
        secondaryTimes : `list` [ `list` [ `int` ] ]
            Nested list with start/end times of the secondary bump tests.
```

Also, although we accept `lowerCamelCase` for variables and class attributes, our preference is for `snake_case`, e.g.; `primaryTimes` -> `primary_times`